lib/models/Site.dart

```diff
@@ -93,7 +93,7 @@ class Site {
     this.rawConfig = rawConfig;
     this.lastManagedUpdate = lastManagedUpdate;
 
-    _updates = EventChannel('net.defined.nebula/$id');
+    _updates = EventChannel('net.defined.nebula/$this.id');
```

Comment:
```suggestion
    _updates = EventChannel('net.defined.nebula/${this.id}');
```

I could be wrong, but I thought this needed to be curly-braced because of the `.`?

Reply:
🤦